### PR TITLE
CLOUDSTACK-9716: Change of network to rvr for baremetal caused vm deployment to get stuck

### PR DESCRIPTION
**Pull request by rashmidixit**
Refer to [CLOUDSTACK-9716](https://issues.apache.org/jira/browse/CLOUDSTACK-9716) for more details

### CODE REVIEW

**Comment by rashmidixit** [comment]
@harikrishna-patnala, @sadhugit Can you please review?

File: systemvm/patches/debian/config/etc/rc.local [hunk 1]
```diff
@@ -42,11 +42,12 @@ then
    echo 1000000 > /proc/sys/net/nf_conntrack_max
 fi
 
-if [ "$router" != "" ]
-then
-    python /opt/cloud/bin/baremetal-vr.py &
+
+if [ "$router" != "" ]; then
+  python /opt/cloud/bin/baremetal-vr.py &
```
Comment (priyankparihar) [inline systemvm/patches/debian/config/etc/rc.local]: @rashmidixit what is the purpose of introducing this change ?

**Comment by SudharmaJain** [comment]
@rashmidixit Could you add a marvin test case? It looks with your changes, master VR Redundant state will be stuck in unknown state.

**Comment by rashmidixit** [comment]
PR #1990 already covers this change. @SudharmaJain @priyankparihar Can you please review #1990. 
Thanks @harikrishna-patnala for the review.

Closing this PR since its already covered.

**Pull request closed without merging**